adapters/flipp/flipp.go

```diff
@@ -0,0 +1,225 @@
+package flipp
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/buger/jsonparser"
+	"github.com/gofrs/uuid"
+	"github.com/prebid/openrtb/v17/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const BannerType = "banner"
+const InlineDivName = "inline"
+const FlippBidder = "flipp"
+const DefaultCurrency = "USD"
```

Comment:
nit:- all these constants can be grouped in one const block for better readability like this :- 

```
const (
	BannerType = "banner"
	InlineDivName = "inline"
	FlippBidder = "flipp"
	DefaultCurrency = "USD"
)
```